libr/cons/dietline.c

```diff
@@ -224,22 +224,30 @@ R_API int r_line_hist_add(const char *line) {
 	if (!I.history.data) {
 		inithist ();
 	}
-	if (I.history.top >= I.history.size) {
-		I.history.top = I.history.index = 0; // workaround
-	}
 	/* ignore dup */
-	if (I.history.index > 0) {
-		const char *data = I.history.data[I.history.index - 1];
+	if (I.history.top > 0) {
```

Comment:
those dietline changes are totally unrelated to your pr, what are you trying to do here?